cabal-haskell.nix.project

```diff
@@ -45,39 +33,18 @@ source-repository-package
 
 ---------- *replace here* ----------------------------------------------------------------------
 
--- You never, ever, want this.
+-- We never, ever, want this.
 write-ghc-environment-files: never
 
 -- Always build tests and benchmarks.
 tests: true
 benchmarks: true
 
--- The only sensible test display option
-test-show-details: direct
```

Comment:
Their 'sensible display option' is wrong. I've already fixed this upstream in Plutus. The reason we use 'direct' is because it gives us colourized output, from things like `tasty`.